drop-boxes/register-all-dropbox/plugin.properties

```diff
@@ -0,0 +1,11 @@
+#
+# Drop box for registering a fastq file as a data set
+#
+# Variables:
+#   incoming-root-dir
+#     Path to the directory which contains incoming directories for drop boxes.
+incoming-dir = ${incoming-root-dir}/QBiC-register-all-data
+incoming-data-completeness-condition = marker-file
+top-level-data-set-handler = ch.systemsx.cisd.etlserver.registrator.api.v2.JavaTopLevelDataSetHandlerV2
+program-class = life.qbic.registration.MainETL
+storage-processor = ch.systemsx.cisd.etlserver.DefaultStorageProcessor
```

Comment:
Does it make sense to separate the variable parts from the invariable ones? E.g.
```suggestion

incoming-data-completeness-condition = marker-file
top-level-data-set-handler = ch.systemsx.cisd.etlserver.registrator.api.v2.JavaTopLevelDataSetHandlerV2
program-class = life.qbic.registration.MainETL
storage-processor = ch.systemsx.cisd.etlserver.DefaultStorageProcessor
# Variables:
#   incoming-root-dir
#     Path to the directory which contains incoming directories for drop boxes.
incoming-dir = ${incoming-root-dir}/QBiC-register-all-data
```

Reply:
Sure, i just sticked to the default layout in the other dropboxes